cosmosdb-scaler/Chart.yaml

```diff
@@ -0,0 +1,35 @@
+# https://helm.sh/docs/topics/charts/#the-chartyaml-file
+
+apiVersion: v2
+name: keda-external-scaler-azure-cosmos-db
```

Comment:
```suggestion
name: keda-add-ons-azure-cosmos-db
```

Reply:
Let's use this name instead of my folder naming suggestion, but the convention is that they are equal so let's align chart & folder name.

Reply:
For sake of consistency, I have renamed the folder to `azure-cosmos-db-add-on` so as to match with `http-add-on`. Will keep the conversation unresolved until we have definitive answer.